ios/Podfile.lock

```diff
@@ -1234,4 +1234,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: bc8161c6bfffeec6e6eaf84be18de5041ddcacf6
 
-COCOAPODS: 1.12.1
+COCOAPODS: 1.11.3
```

Comment:
Can you make sure to use `bundle exec pod install`? We are trying to use the Gemfile to standardize the pod version ([context](https://github.com/Expensify/App/issues/21318)).

Reply:
Ough, sorry 😬